srcpkgs/netatalk/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'netatalk'
+pkgname=netatalk
+version=3.1.15
+revision=1
+archs='x86_64'
+build_style=gnu-configure
+conf_files="
+	/etc/dbus-1/system.d/netatalk-dbus.conf
+	/etc/extmap.conf
+	/etc/afp.conf
+	/etc/dbus-session.conf"
```

Comment:
Taking over such generic file by random package look like bad idea, especially as current xbps will overwrite if it existed. Maybe install it in /usr/share/examples?

Reply:
The "dbus-session.conf" can be relocated, other files should be fine. FreeBSD add suffix ".sample" to avoid the file conflict:

```
# pkg info -lx netatalk | grep -e etc -e share
        /usr/local/etc/afp.conf.sample
        /usr/local/etc/extmap.conf.sample
        /usr/local/etc/pam.d/netatalk.sample
        /usr/local/etc/rc.d/netatalk
        /usr/local/share/aclocal/netatalk.m4
        /usr/local/share/licenses/netatalk3-3.1.15,1/GPLv2
        /usr/local/share/licenses/netatalk3-3.1.15,1/LICENSE
        /usr/local/share/licenses/netatalk3-3.1.15,1/catalog.mk
```

> especially as current xbps will overwrite if it existed

I edited the afp.conf and then reinstalled the package, the afp.conf is not overwrote.

Reply:
> afp.conf is not overwrote

OK, sorry, new installs are fine and overwrite happens in slightly different conditions.